File: 1295. Find Numbers with Even Number of Digits/solution.py
class Solution(object):
  def findNumbers(self, nums):
    """
    :type nums: List[int]
    :rtype: int
    """
    count = 0
    for num in nums:
      if (num > 9 and num < 100) or (num > 999 and num < 10000) or num == 100000:
        count += 1
        
    return count

File: 1295. Find Numbers with Even Number of Digits/solution.cpp
class Solution {
public:
  int findNumbers(vector<int>& nums) {
    int count = 0;
    for (int index = 0; index < nums.size(); index++){
      int num = nums[index];
      if ((num > 9 && num < 100) || (num > 999 && num < 10000) || num == 100000){
        count += 1;
      }
    }

    return count;
  }
};